src/css/archwizard.scss

```diff
@@ -1,11 +1,127 @@
+//
+// 'Null' colors
+// -------------
+//
+// You can use `null` instead of a color value to exclude the corresponding property from CSS output.  For example,
+//
+//   $wz-colors: (
+//     '_': (
+//       'editing': (
+//         'border-color-default': null,
+//         'border-color-hover': null
+//       )
+//     )
+//   )
+//
+//   will turn off red border for wizard steps in 'editing' state.
+//
+// Default colors
+// --------------
+//
+// You can set default colors by using '_' in place of an indicator style or a wizard step state.
+// The `wz-get-color` function looks up the requested color in the following order, picking the first defined entry:
+//
+// - $wz-colors[$indicator-style][$step-state][$color]
+// - $wz-colors[$indicator-style]['_'][$color]
+// - $wz-colors['_'][$step-state][$color]
+// - $wz-colors['_']['_'][$color]
+//
+$wz-colors: (
+  '_': (
+    'default': (
+      'border-color-default': #E6E6E6,
+      'background-color-default': #E6E6E6,
+      'symbol-color-default': #E6E6E6,
+      'border-color-hover': darken(#E6E6E6, 10%),
+      'background-color-hover': darken(#E6E6E6, 5%),
+      'symbol-color-hover': darken(#E6E6E6, 10%),
+    ),
+    'current': (
+      'border-color-default': #808080,
+      'background-color-default': #808080,
+      'symbol-color-default': #808080,
+      'border-color-hover': darken(#808080, 10%),
+      'background-color-hover': darken(#808080, 5%),
+      'symbol-color-hover': darken(#808080, 10%),
+    ),
+    'done': (
+      'border-color-default': #339933,
+      'background-color-default': #339933,
+      'symbol-color-default': #339933,
+      'border-color-hover': darken(#339933, 10%),
+      'background-color-hover': darken(#339933, 5%),
+      'symbol-color-hover': darken(#339933, 10%),
+    ),
+    'optional': (
+      'border-color-default': #38ef38,
+      'background-color-default': #38ef38,
+      'symbol-color-default': #38ef38,
+      'border-color-hover': darken(#38ef38, 10%),
+      'background-color-hover': darken(#38ef38, 5%),
+      'symbol-color-hover': darken(#38ef38, 10%),
+    ),
+    'editing': (
+      'border-color-default': #FF0000,
+      'background-color-default': #FF0000,
+      'symbol-color-default': #FF0000,
+      'border-color-hover': darken(#FF0000, 10%),
+      'background-color-hover': darken(#FF0000, 5%),
+      'symbol-color-hover': darken(#FF0000, 10%),
+    ),
+  ),
+  'large-filled-symbols': (
+    '_': (
+      'symbol-color-default': black,
+      'symbol-color-hover': black,
+    )
+  )
+) !default;
+
+// line color
+$wz-line-color: #E6E6E6 !default;
+
+// label colors
+$wz-label-color-default: #808080 !default;
+$wz-label-color-hover: darken($wz-label-color-default, 20%) !default;
 
 // dot definitions
+//
+// Here and below, "dots" are wizard step indicators represented as circles in the navigation bar.
+// Depending on step indicator style selected for the wizard, dots can either be small (indicator style 'small')

```

Comment:
Depending on **the** step indicator style...

Reply:
Fixed in ab5bfe9